docs/user_stories/capacity_calculation_rsc.md

```diff
@@ -0,0 +1,119 @@
+
+```java
+LoadFlowParameters loadFlowParameters = new LoadFlowParameters().setVoltageInitMode(LoadFlowParameters.VoltageInitMode.DC_VALUES);
+LoadFlowResult result = loadFlow.run(VariantManagerConstants.INITIAL_VARIANT_ID, loadFlowParameters).join();
+```
+<br />
+
+<img src="./images/Compute_Sensitivity.svg" style="vertical-align: bottom"/>
+The sensitivity computation module is dedicated to compute linearized impact of network small variations on some elements state variables. The sensivity computation is fully described [here](../sensitivity/index.md). In this user story, we use this module to compute all coefficients of the cost function. The model behind is Hades2, which is a non open source software, but available in a freeware mode for experimental purposes. For more details, please visit this [page](https://rte-france.github.io/hades2/features/loadflow.html) to learn about Hades2.
+
+<br />
+
+<img src="./images/Modify_iAL.svg" style="vertical-align: bottom"/>
+Remedial actions are read from the CRAC file and given to the optimizer which is a specific module. The best set of remedial actions is converted in actions understandable by PowSyBl framework. The CRAC file provides also the contingency list, which is also converted in an understandable object for PowSyBl framework called [Contingency](../contingencies/index.md).
+
+```java
+ContingenciesProvider contingenciesProvider = new ContingenciesProvider() {
+            @Override
+            public List<Contingency> getContingencies(Network network) {
+                  // Code here how you want to fill/map the list of contingencies.
+            }
+        };
+```
+
+<br />
+
+<img src="./images/Compute_SA.svg" style="vertical-align: bottom"/>
+The final set of remedial actions is validated through a security analysis. A security analysis is run needs an input variant, a set of parameters as a `securityAnalysisParameters` object and a set of contingencies as a `contingenciesProvider` object.
+
+```java
+SecurityAnalysis securityAnalysis = new Hades2SecurityAnalysisFactory().create(networkBe, computationManager, 0);
+SecurityAnalysisParameters securityAnalysisParameters = new SecurityAnalysisParameters(); // Default parameters.
+network.getVariantManager().cloneVariant(VariantManagerConstants.INITIAL_VARIANT_ID, "saVariant");
+SecurityAnalysisResult securityAnalysisResult = securityAnalysis.run("saVariant", securityAnalysisParameters, contingenciesProvider).join();
+```
+
+<br />
+
+Note that the data management is handled by the PowSyBl feature called Application File System (AFS).
+
+# External features are:
+
+<img src="./images/Compute_Optimizer.svg" style="vertical-align: bottom"/>
+The cost function builder is in fact a big toolbox using some power system blocks from PowSyBl framework. For more details about this builder, please refer to [FARAO website](https://farao-community.github.io/). The optimization is based on a OR-Tools solution.
```

Comment:
Instead of 

> The optimization is based on a OR-Tools solution.

Say:

> OR-Tools library is used to perform the optimization (https://developers.google.com/optimization/)